push/src/main/java/org/aerogear/mobile/push/UnifiedPushConfig.java

```diff
@@ -0,0 +1,30 @@
+package org.aerogear.mobile.push;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class UnifiedPushConfig {
+
+    private String alias;
+    private List<String> categories;
+
+    public String getAlias() {
+        return alias;
+    }
+
+    public void setAlias(String alias) {
+        this.alias = alias;
+    }
+
+    public List<String> getCategories() {
+        if (categories == null) {
+            return new ArrayList<>();
+        }
+        return categories;
+    }
+
+    public void setCategories(List<String> categories) {
+        this.categories = categories;
```

Comment:
Never assign a mutable to an internal variable to avoid losing control about its content.
Consider copying the list.